README.md

```diff
@@ -51,27 +51,49 @@ There will be a grand prize winner out of all projects as well as a 'Best Intell
 We'd love to hear about your project, tell us what you are working on!
 
 ## Samples to get inspired ✨
+
 If you are looking for scenarios or project ideas to get inspired, we have many samples available for you to explore! Check out below repositories to discover what you can build and get some inspiration for your hack!
 
-- [ConsoleGPT](https://github.com/aaronpowell/ConsoleGPT) - .NET Console demos for a chat experience using Azure OpenAI
-- [Image Generation sample](https://github.com/aaronpowell/GenerateAndChill) - Generate images with DALL-E, it also supports non-English language prompts through translations
-- [Chat with your own data](https://github.com/Azure-Samples/azure-search-openai-demo-csharp) - An application using the RAG (Retrival Augmented Generation) pattern to allow you to ask questions of your own data
+### AI
+
+* [ConsoleGPT](https://github.com/aaronpowell/ConsoleGPT) - .NET Console demos for a chat experience using Azure OpenAI
+* [Image Generation sample](https://github.com/aaronpowell/GenerateAndChill) - Generate images with DALL-E, it also supports non-English language prompts through translations
+* [Chat with your own data](https://github.com/Azure-Samples/azure-search-openai-demo-csharp) - An application using the RAG (Retrival Augmented Generation) pattern to allow you to ask questions of your own data
+
+### Cloud Native
+
+* [eShop on Containers](https://github.com/dotnet-architecture/eShopOnContainers)
 
 ## Recommended Learning Materials 📚
 
+### AI
```

Comment:
Should we add https://aka.ms/genai-beginners as well?

Reply:
Yup, adding it.